plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/inline/InlineChatEditorHint.kt

```diff
@@ -68,7 +67,7 @@ class InlineChatEditorHint {
         coloredText.appendToComponent(component)
         val shortcutComponent = HintUtil.createInformationComponent()
         val shortCut = KeymapUtil.getShortcutText("aws.toolkit.jetbrains.core.services.cwc.inline.openChat")
-        if (!SystemInfo.isWindows && shortCut == "⌃I") {
+        if (shortCut == "⌘I") {
```

Comment:
please fix this properly by building the popup based on the user shortcut

Reply:
sure I can remove this

Reply:
Why are we using a hardcoded shortcut icon? Will this never show an icon in Windows or Linux? This does not feel like an adequate experience if it only supports a default case on one OS.

Reply:
removed

Reply:
it was a styling decision